src/userSettingsMapping.js

```diff
@@ -30,6 +30,7 @@ const settingsKeyMapping = {
     email: {
         label: i18n.t('E-mail'),
         type: 'emailModal',
+        validators: ['email'],
```

Comment:
adding this back so that v41 email field gets validated. This will also complement the v42 email modal field validators, but I'm assuming (hoping) that email validation is relatively straightforward such that they're going to produce incompatible results (the validators come from different dhis2 ui libraries)